src/components/atoms/UiBackdrop/UiBackdrop.stories.ts

```diff
@@ -0,0 +1,33 @@
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import { UiBackdrop } from '@/../index';
+
+const meta = {
+  title: 'Atoms/Backdrop',
+  component: UiBackdrop,
+  decorators: [ () => ({ template: '<div class="min-h-80"><story/></div>' }) ],
+} satisfies Meta<typeof UiBackdrop>;
+export default meta;
+type Story = StoryObj<typeof UiBackdrop>;
+export const Basic: Story = {
+  render: () => ({
+    components: { UiBackdrop },
+    template: '<UiBackdrop />',
+  }),
+};
+Basic.parameters = {
+  chromatic: { disableSnapshot: false },
+  docs: {
+    source: {
+      code: `<template>
```

Comment:
<img width="1010" alt="Zrzut ekranu 2023-03-13 o 21 01 00" src="https://user-images.githubusercontent.com/12138170/224818998-c23a95e2-9c39-4948-96d9-9877d734d5e9.png">
I love this approach to Source code; But we should think how to do it smarter. I will thing about that in this PR :D later